modules/nw-cluster-mtu-change-about.adoc

```diff
@@ -0,0 +1,77 @@
+
+- At least one rolling reboot is required to complete the migration to a new MTU. During this time, some nodes are not available as they restart.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R180
+- Specific applications deployed to the cluster with shorter timeout intervals than the absolute TCP timeout interval might experience disruption during the change.
+
+// https://github.com/openshift/enhancements/pull/926/files#diff-166e452788d3711a8dfa7254d867f156c7b7c659f172e633acbf6852504ae358R190
+- Different endpoints in the cluster might use different MTUs during the migration. As a result it is possible that some network traffic is dropped until the MTU change is finalized.
+
+[id="mtu-value-selection_{context}"]
+== MTU value selection
+
+When planning your MTU migration there are two related but distinct MTU values to consider.
+
+* *Hardware MTU*: This MTU value is set based on the specifics of your network infrastructure.
+* *Cluster network MTU*: This MTU value is always less than your hardware MTU to account for the cluster network overlay overhead. The specific overhead is determined by your cluster network provider:
+** *OVN-Kubernetes*: `100` bytes
+** *OpenShift SDN*: `50` bytes
+
+// modules/nw-operator-cr.adoc
+If your cluster requires different MTU values for different nodes, you must set this value to `100` less than the lowest MTU value in your cluster. For example, if some nodes in your cluster have an MTU of `9001`, and some have an MTU of `1500`, you must set this value to `1400`.
+
+[id="how-the-migration-process-works_{context}"]
+== How the migration process works
+
+The following table summarizes the migration process by segmenting between the user-initiated steps in the process and the actions that the migration performs in response.
+
+.Live migration of the cluster MTU
+[cols="1a,1a",options="header"]
+|===
+
+|User-initiated steps|{product-title} activity
+
+|
+Set the following values in the Cluster Network Operator configuration:
+
+- `spec.migration.mtu.network.from`
+- `spec.migration.mtu.network.to`
+- `spec.migration.mtu.machine.to`
+
+|
+*Cluster Network Operator (CNO)*: Confirms that each field is set to a valid value.
+
+- The `mtu.network.from` field must equal `network.status.clusterNetworkMTU` which is the current MTU of the cluster network.
+- If the `mtu.network.to` field is specified, it must allow for the overlay overhead for the cluster network provider. For OVN-Kubernetes, this value is `100` and for OpenShift SDN this value is `50`.
+
+If the values provided are valid, the CNO writes out a new temporary configuration with the MTU for the cluster network set to the value of the `mtu.network.to` field.
+
+*Machine Config Operator (MCO)*: Performs a rolling reboot of each node in the cluster.
+
+|Reconfigure the MTU of the primary network interface for the nodes on the cluster.
+|N/A
```

Comment:
Perhaps we can add a small comment here on the options with which this might be achieved, some of them are:
* Deploying an updated ifcfg or NM profile with the MTU throuh MCO
* Changing the DHCP server MTU setting
* Changing boot parameters (if possible)

Reply:
Added.